lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -273,6 +277,7 @@ def __init__(self, runbook: schema.Platform) -> None:
         super().__init__(runbook=runbook)
         self._eligible_capabilities: Dict[str, List[AzureCapability]] = {}
         self._locations_data_cache: Dict[str, AzureLocation] = {}
+        self.data_disks: List[DataDiskSchema] = []
```

Comment:
Don't put data_disks to a variable of platform. The platform is used to deploy multiple envs at the same time. And the variable is used in one place, the returned value from `_generate_data_disks` is ok.

Reply:
removed